src/launchWizard/launchWizard.ts

```diff
@@ -152,38 +152,7 @@ async function openFilePicker(description) {
       title: description,
     })
     .then(async (fileUri) => {
-      if (fileUri && fileUri[0]) {
```

Comment:
Github won't let me comment on earlier lines. 

Line 139 - it's fine for the picker to start with the workspace folder, but it needs to be able to point at any folder on the machine. E.g., I've setup classpaths to refer to jars under ~/.ivy2/local/.... which is where they get put for a dependent schema by 'sbt publishLocal'. 

So navigating up and out of the workspace is needed.

Reply:
So you should be able to navigate out of the initial folder to any folder you want, I believe I have moved out of the initial folder many times. Are you not able to do that currently?

Reply:
I have not tried it in a while.

Reply:
Oh okay it might worth while to try it out on this PR just to ensure if this is still an issue for you.

Reply:
To do that I need to up my game on daffodil-vscode.

I need to know how to build and deploy it locally based on a git sandbox, and how to tell it to use the local sandbox daffodil snapshot instead of using a released one. 

Presumably you guys who are actively developing on this do this all the time, but I have yet to do this. 

Instructions are where?

Reply:
So some instructions for debugging it locally can be found here [Development Wiki Page](https://github.com/apache/daffodil-vscode/wiki/Apache-Daffodil%E2%84%A2-Extension-for-Visual-Studio-Code:-Development#apache-daffodil-extension-for-visual-studio-code-development). 

However, there is a decently quick way to get up running. After you meet the pre-reqs listed on the link above and get this PR pulled down, run these commands at the root of repo (this is described in the README.md under section [Package VSIX and Debugger](https://github.com/apache/daffodil-vscode#package-vsix-and-debugger) and [Installation From File](https://github.com/apache/daffodil-vscode#installation-from-file)):

```bash
rm -rf node_modules out || true
yarn install
yarn package
code --install-extension apache-daffodil-vscode-1.3.0-SNAPSHOT.vsix
```

So what this does is:

- Remove any hanging installed node package or compiled code
- Install the latest version of the node packages
- Compile the extension and debugger code, into an extension file (`.vsix`)
- Then installs the just built extension file into your VS Code editorer.

Now you should have the extension installed so you can open up VS Code, open the launch config wizard and test if you can navigate out of the initial folder (it should use your native os file explorer window). 

However, if this is also to much I could upload a VSIX to this PR. This would allow to download and install the vsix file made from this PRs code, then test out the navigation.

Reply:
@mbeckerle Am I good to merge this or do you have updates on testing the browsing?

Reply:
I have setting this up on my todo list. I have not had any time to do this.

Reply:
Oh okay no problem, do you want me to keep this open till you get a chance to do that?

Reply:
To be clear. You are good to merge.

Reply:
Alright thank you!